javascript-arrays/main.js

```diff
@@ -0,0 +1,32 @@
+var colors = [
+  'red',
+  'white',
+  'blue'
+];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+console.log('America is' + ' ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is' + ' ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.');
+
+console.log('value of colors:', colors);
+
+var students = [
+  'Leonardo',
+  'Donatello',
+  'Raphael',
+  'Michelangelo'
+];
+
+var numberOfStudents = students.length;
+console.log('There are ' + numberOfStudents + ' students in the class.');
+
+var lastIndex = students[3];
```

Comment:
What if the array was longer or shorter? You should determine the last index programmatically.